qa/t/test_shutdown.t

```diff
@@ -1,72 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
what's happing here?

if the shutdownhtml is no longer supported then all the related code should be removed.
if it is still supported then this test should remain.